code/game/objects/lighting/rogue_fires.dm

```diff
@@ -343,127 +343,117 @@
 			if(!pot.reagents.has_reagent(/datum/reagent/water, 33))
 				to_chat(user, "<span class='notice'>Not enough water.</span>")
 				return TRUE
-			if(pot.reagents.chem_temp < 374)
-				to_chat(user, "<span class='warning'>[pot] isn't boiling!</span>")
-				return
 			if(istype(W, /obj/item/reagent_containers/food/snacks/produce/oat))
 				if(do_after(user,2 SECONDS, target = src))
 					user.visible_message("<span class='info'>[user] places [W] into the pot.</span>")
 					qdel(W)
 					playsound(src.loc, 'sound/items/Fish_out.ogg', 20, TRUE)
-					pot.reagents.remove_reagent(/datum/reagent/water, 32)
-					sleep(300)
+					sleep(1000)
```

Comment:
these are massive time increases in alot of places from 30 seconds to 100 seconds

Reply:
> these are massive time increases in alot of places from 30 seconds to 100 seconds

That's because I removed the requirement to wait for the water to start boiling since it was being a pain in random cases. Making it consistent also just makes more sense. Certain soups which were functionally identical had varied cook times for seemingly zero reason

Reply:
> not your fault but this code is so bad in general

Yeah, it needs a full rewrite but I was too tired to go through the hassle so I'm settling for "Functional" in case a full rewrite couldn't be finished before the next test